terraform/modules/tf_stream_alert_athena/variables.tf

```diff
@@ -37,3 +37,8 @@ variable "athena_data_buckets" {
 variable "prefix" {
   type = "string"
 }
+
+variable "refresh_interval" {
+  type    = "string"
+  default = "rate(10 minutes)"
```

Comment:
Want to confirm that is it 10 or 1 minutes for cloudwatch event refresh interval?  In PR description, it mentions `This also modifies how Lambda is invoked, since it now runs on a CloudWatch event timer of 1 minute`.